lib/zoom/actions/token.rb

```diff
@@ -8,12 +8,15 @@ module Token
       post 'access_tokens',
         '/oauth/token?grant_type=authorization_code',
         base_uri: 'https://zoom.us/',
-        require: %i[code redirect_uri]
+        require: %i[code redirect_uri],
+        args_to_params: { auth_code: :code },
+        headers: { 'Content-Type' => 'application/x-www-form-urlencoded' }
```

Comment:
I would define `{ 'Content-Type' => 'application/x-www-form-urlencoded' }` in a constant such as `FORM_URLENCODED_HEADER`.